### PR TITLE
Add new startups page

### PR DESCRIPTION
**Pull request by inesoaresilva**
### Context
#2047 

🎨 [ **Design in Sketch** ](https://www.sketch.com/s/9d642daa-3698-4c49-b32b-14feec941f8e/a/5nnmjyd/r/wokadE#Inspect)

🔗 [**Preview Link**](https://deploy-preview-2049--objective-northcutt-37494c.netlify.app/startups/)

### CODE REVIEW

**Comment by davidhajba** [comment]
hey @inesilva 
could you please add a preview link?
under the 'Deploy Preview ready!' I see the Mainmatter homepage

**Comment by inesoaresilva** [comment]
> hey @inesilva could you please add a preview link? under the 'Deploy Preview ready!' I see the Mainmatter homepage

Yup, I just updated the description

**Review by oscard0m — CHANGES_REQUESTED** [review]
The page looks good!

Check Percy. The header has a duplicated link to 'Blog':
image

File: src/startups.njk [hunk 1]
```diff
@@ -0,0 +1,108 @@
+---js
+{
+  title: "Startups",
+  description: "Find out how can we help you grow your business.",
```
Comment (marcoow) [inline src/startups.njk]: I'm not sure this works as a description – can you help @SaeLLo? See [this](https://ahrefs.com/blog/open-graph-meta-tags/) for reference

File: src/startups.njk [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+{% from "post-card.njk" import postCard %}
+{% from "nav-pagination.njk" import navPagination %}
+
+
+{% set 'content' = {
+  "title": "Burst through your scale-up bottleneck. We'll be your <em>expert</em> guides.",
+  "image": "/assets/images/photos/ship-containers.jpg",
+  "text": "No business has “make or break” moments like a startup. At Mainmatter, we can instantly scale-up your team to develop digital products.
+   Our experts teach, cross-pollinate and collaborate with our clients for results that go beyond the scope of the project.",
+  "alt": "A massive vessel packed with cargo containers navigating in a water channel",
```
Comment (marcoow) [inline src/startups.njk]: This is the Ever Given in the Suez Canal: https://en.wikipedia.org/wiki/Ever_Given#2021_Suez_Canal_grounding

File: src/startups.njk [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  "linkUrl": "/contact/",
+  "linkText": "Get started with us today",
+  "image": "/assets/images/photos/mainmatter-team-smiles-blur.jpg",
+  "alt": "Team member leaning against wall taking notes while talking to three other team members",
+  "loading": "lazy"
+} %}
+{{ colorHero('center', 'large', 'black', content) }}
+
+<h3 class="success_stories__title">Discover our success stories</h3>
+{% set logoCompanies = ['Qonto', 'Trainline', 'Cardstack', 'Timify', 'Experteer', 'Robin Hood', 'Dim3' ] %}
```
Comment (marcoow) [inline src/startups.njk]:
```suggestion
{% set logoCompanies = ['Qonto', 'Trainline', 'Cardstack', 'Timify', 'Experteer', 'Rail Europe', 'Sage' ] %}
```

File: src/startups.njk [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+  "text": "Scale your team instantly and sustainably. We’ll work with your existing team to develop products, productivity, or best practices so you can keep growing, even after our collaboration ends.",
+  "linkUrl": "/services/team-augmentation-and-training/",
+  "linkText": "Get started with us today"
+} %}
+
+{% set 'slide_2' = {
+  "subtitle": "Digital Product Development Support",
+  "color": "white",
+  "title": "The shortcut on your roadmap",
+  "text": "We will begin by setting up a web server for our web application using the Axum framework, which is built on top of Tokio and Hyper. We will learn about the FromRequest and IntoResponse traits, Extractors, Routers, and how to connect JSON serialization and deserialization. If time permits, we will also create a web socket connection for our clients.",
```
Comment (marcoow) [inline src/startups.njk]: wrong text

File: src/startups.njk [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+  <h5 class="posts__title h5">Read more about our work</h5>
+  <ol class="post-cards">
+    {%- for post in pagination.items -%}
+      <li class="post-cards__item">
+        {{ postCard(post) }}
+      </li>
+    {%- endfor -%}
+  </ol>
+  {% if pagination.pageNumber === 0 %}{% set nextLabel = 'Explore more articles' %}{% else %}{% set nextLabel = 'Next' %}{% endif %}
+  {{ navPagination(pagination.href.previous, pagination.href.next, nextLabel) }}
```
Comment (marcoow) [inline src/startups.njk]:
```suggestion
```

no pagination here

**Review by marcoow — COMMENTED** [review]
Nice! Just some comments on minor aspects

File: src/startups.njk (hunk 4 above)
Comment (SaeLLo) [reply src/startups.njk]: -> "We’ve got a proven track record as digital product developers. We collaborate with our clients from the strategy phases to validation to MVP and beyond.  Our experienced team will get you to market."

**Review by SaeLLo — CHANGES_REQUESTED** [review]

File: src/startups.njk (hunk 3 above)
Comment (inesoaresilva) [reply src/startups.njk]: @davidhajba do you have rail europe logo in monochrome version?

**Review by SaeLLo — APPROVED** [review]

**Review by oscard0m — CHANGES_REQUESTED** [review]
I see in Percy there are some pages generated which I think we don't want to:
- /startups/9/index.html
- /startups/8/index.html
- /startups/7/index.html
- ...

I guess is related to pagination?

Also, CI failing now.

**Comment by davidhajba** [comment]
It looks great, 

* Background images 'Instantly add years of expertise' & 'Grow your business with us' 
looks bad quality on my side.

* I'm not sure about the extra black transparent overlay animation on the top, I didn't suggest anything like this, the pictures are colored as a final result, and no further overlay is needed.

**Comment by inesoaresilva** [comment]
> It looks great,
> 
> * Background images 'Instantly add years of expertise' & 'Grow your business with us'
>   looks bad quality on my side.
> * I'm not sure about the extra black transparent overlay animation on the top, I didn't suggest anything like this, the pictures are colored as a final result, and no further overlay is needed.

I removed the black overlay, and the photos are looking better now, I should be exporting the images with 2x the size, instead of the standard 1x (otherwise, they don't get enough pixel density to look good on retina screens)

File: src/startups.njk (hunk 1 above)
Comment (inesoaresilva) [reply src/startups.njk]: @SaeLLo do you have any updates on this?

**Comment by davidhajba** [comment]
thank you  @inesilva 
* looks much better now
* yes, retina screens always need 2x size when importing pictures from Sketch to achieve the same resolution you see
* this double size picture should work on bigger screens too (big enough to look good) but on the website it still looks too low quality
* as I know there's some automatic tool 'optimizes' picture size, but imo it doesn't work well, if the UX impression is low quality images, is it possible to improve this?

File: src/startups.njk (hunk 1 above)
Comment (SaeLLo) [reply src/startups.njk]: @inesilva Where is this description?

Comment (SaeLLo) [reply src/startups.njk]: Let's use: 
Burst through your scale-up bottleneck. We'll be your expert guides.

File: src/startups.njk [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+{
+  title: "Startups",
+  description: "Find out how can we help you grow your business.",
+  layout: "base.njk",
+  pagination: {
+    data: "collections.posts",
+    size: 3,
+    alias: "posts",
+    before: function(data) { return data.slice(1); }
+  },
```
Comment (oscard0m) [inline src/startups.njk:11]: I suspect this is copy-paste and is generating the undesired pagination in Percy, could be? @inesilva

Comment (oscard0m) [inline src/startups.njk]:
```suggestion
```

I suspect this is copy-paste and is generating the undesired pagination in Percy, could be? @inesilva

**Review by oscard0m — CHANGES_REQUESTED** [review]

File: src/startups.njk (hunk 6 above)
Comment (inesoaresilva) [reply src/startups.njk]: This is required to display the posts at the end of the page:
image

Comment (oscard0m) [reply src/startups.njk]: > This is required to display the posts at the end of the page: image

Oh! I see. 

Then the question here is: What should we do here?
a) We only show the first three blog posts, and the link points to `/blog/page/2/#posts`
b) What we have right now is correct.

Comment (davidhajba) [reply src/startups.njk]: I think we're missing the 'Explore more articles' button from the bottom, 
Imo if you could add it, the current version would be good

Screenshot 2023-06-01 at 12 35 45

Comment (inesoaresilva) [reply src/startups.njk]: which link @oscard0m ?

Comment (inesoaresilva) [reply src/startups.njk]: the one of explore more articles?

Comment (oscard0m) [reply src/startups.njk]: > which link @oscard0m ?

I thought we would have this on the homepage:
image

So... forget about the link part a)/b). We need to find a way to render the first 3 blog posts but to not have pagination.

Comment (inesoaresilva) [reply src/startups.njk]: pagination removed ✅

File: src/startups.njk (hunk 1 above)
Comment (inesoaresilva) [reply src/startups.njk]: @marcoow @SaeLLo I updated the text. Let me know if it looks good or if you have other suggestions

**Review by oscard0m — CHANGES_REQUESTED** [review]
Check Percy's comments. I see a padding/margin issue at the end of some pages.

**Review by SaeLLo — APPROVED** [review]

**Comment by inesoaresilva** [comment]
@SaeLLo the description it is here - https://github.com/mainmatter/mainmatter.com/pull/2049/commits/91df17f9d234ad04f93c8531c1d29e294cc8aa0d

**Review by oscard0m — APPROVED** [review]

**Comment by SaeLLo** [comment]
@inesilva and @oscard0m This is ready, right?

**Comment by oscard0m** [comment]
> @inesilva and @oscard0m This is ready, right?

@inesilva is on top of this. On my side, LGTM

**Comment by SaeLLo** [comment]
@inesilva Can we please get this done? What is missing?

**Comment by inesoaresilva** [comment]
> @inesilva Can we please get this done? What is missing?

All done, I just merged it

**Pull request merged**